central/graphql/resolvers/gen/main.go

```diff
@@ -90,12 +90,6 @@ var (
 				ParentType: reflect.TypeOf(storage.Node{}),
 				FieldName:  "Scan",
 			},
-			// TODO(ROX-6194): Remove this entirely after the deprecation cycle started with the 55.0 release.
```

Comment:
I'd suggest removing everything related to ROX-6194 in one PR and close the ticket. Can happen here or separately.

Reply:
Everything related to ROX-6194 has already been removed except this,

```
message Policy {
...
reserved                                            10; //was repeated Exclusion whitelists
...
}
```

Reply:
I also see mentions in:
* `pkg/env/baseline.go`
* `qa-tests-backend/src/test/groovy/RiskTest.groovy`